### PR TITLE
getMeetingData and getMeeting reduction

### PR DESCRIPTION
**Pull request by rrballenger17**

### CODE REVIEW

File: src/com/comeet/UserService.java [hunk 1]
```diff
@@ -120,33 +121,37 @@
...
      * @throws Exception On an unexpected error.
      */
-    @POST
-    @Path("/{orgDomain}/meetings/attendees")
+    @GET
+    @Path("/{orgDomain}/meeting/data")
     @Produces("application/json")
-    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
-    public Attendees getAttendees(@Context HttpHeaders headers,
+    public Meeting getAttendees(@Context HttpHeaders headers,
```
Comment (Dairai) [inline src/com/comeet/UserService.java]: Method name should be more applicable e.g. getMeetingData

File: src/com/comeet/UserService.java [hunk 2]
```diff
@@ -120,33 +121,37 @@
...
                     @PathParam("orgDomain") String orgDomain, 
-                    @DefaultValue("") @FormParam("id") String id)
+                    @DefaultValue("") @QueryParam("id") String id)
                     throws ServiceResponseException, Exception {
 
+        id = URLDecoder.decode(id, "UTF-8");
+        id = id.replaceAll(" ", "+");
+        
+        System.out.println("ID " + id);
```
Comment (Dairai) [inline src/com/comeet/UserService.java]: How come you are printing the id to the console?

**Review by Dairai — CHANGES_REQUESTED** [review]
Need to have a more appropriate method name for getAttendees

**Comment by rrballenger17** [comment]
rename and remove print  - complete

**Review by Dairai — APPROVED** [review]

**Pull request merged**